app/models/foreman_openscap/arf_report.rb

```diff
@@ -5,6 +5,7 @@ class ArfReport < ::Report
     include Taxonomix
     include OpenscapProxyExtensions
 
+    # attr_accessible :host_id, :reported_at, :status, :metrics
```

Comment:
When trying to upload a report: ActiveModel::MassAssignmentSecurity::Error: Can't mass-assign protected attributes: host, reported_at, status, metrics

We will need to add openscap_proxy in there ass well